### PR TITLE
Meeting Notes: 2018-04-19

### LINKED ISSUE #292: Node.js Foundation Community Committee Meeting 2018-04-19
## Time

**UTC Thu 19-Apr-2018 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Thu 19-Apr-2018 09:00 (09:00 AM) |
| US / Mountain | Thu 19-Apr-2018 10:00 (10:00 AM) |
| US / Central  | Thu 19-Apr-2018 11:00 (11:00 AM) |
| US / Eastern  | Thu 19-Apr-2018 12:00 (12:00 PM) |
| London        | Thu 19-Apr-2018 17:00 (05:00 PM) |
| Amsterdam     | Thu 19-Apr-2018 18:00 (06:00 PM) |
| Moscow        | Thu 19-Apr-2018 19:00 (07:00 PM) |
| Chennai       | Thu 19-Apr-2018 21:30 (09:30 PM) |
| Hangzhou      | Fri 20-Apr-2018 00:00 (12:00 AM) |
| Tokyo         | Fri 20-Apr-2018 01:00 (01:00 AM) |
| Sydney        | Fri 20-Apr-2018 02:00 (02:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Community%20Committee+Meeting+2018-04-19&iso=20180419T16
* or http://www.wolframalpha.com/input/?i=04PM+UTC%2C+Apr+19%2C+2018+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1-PpcKo6uksDDbqWFHGGq1vycOW8OjMICY1S1L6-9HYU/edit>

## Agenda

Extracted from **cc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### User Feedback
* 2018-04-27 Public User Feedback Meeting - Enterprise Use Case [#52](https://github.com/nodejs/user-feedback/issues/52)

### nodejs/admin

* Travel Fund approval and being careful about funds available [#99](https://github.com/nodejs/admin/issues/99)
* Add Node Slack Community to scope [#49](https://github.com/nodejs/admin/pull/49)

### nodejs/community-committee

* CommComm Membership: Call For Feedback on New Definitions [#276](https://github.com/nodejs/community-committee/issues/276)

### nodejs/mentorship

* We need more active members of the WG [#32](https://github.com/nodejs/mentorship/issues/32)
* Set a Code of Conduct for mentors and mentees [#31](https://github.com/nodejs/mentorship/issues/31)
* Create a mentorship subgroup focused on to identify 6 month projects in Node.js [#30](https://github.com/nodejs/mentorship/issues/30)

## Invited

* CommComm Members: @nodejs/community-committee

### Observers
* @DiegoZoracKy (Diego ZoracKy - observer)
* @ZibbyKeaton (Zibby Keaton - observer)
* @Bamieh (Ahmad Bamieh - observer)
* @pchrysa (Chrysa - observer)
* @foadlab (observer)
* @codeekage (Abraham Agiri - observer)
* @Voskan (Voskan Voskanyan - observer)
* @waleedashraf (Waleed Ashraf - observer)
* @maddhruv (Dhruv Jain - observer)
* @elainechan (Elaine Chan - observer)

## Notes

The agenda comes from issues labelled with `cc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: Will be added a few minutes before meeting starts
* For those who just want to watch: 
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

### PR DESCRIPTION
**Pull request by JemBijoux**
Closes #292

### CODE REVIEW

File: meetings/2018-04-19.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+## Present
+
+- @bnb (Tierney Cyren - member)
+- @JemBijoux (Jem Bezooyen - member)
+- @chowdhurian (Manil Chowdhury - member)
+- @waleedashraf (Waleed Ashraf - observer)
+- @amiller-gh (Adam Miller - member)
+- @dshaw (Dan Shaw - member)
+- @hackygolucky (Tracy Hinds)
```
Comment (bnb) [inline meetings/2018-04-19.md]: Tracy is a member 😄

File: meetings/2018-04-19.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+- @mdawson (Michael Dawson)
```
Comment (bnb) [inline meetings/2018-04-19.md]: Michael is also a member 😄

File: meetings/2018-04-19.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+- @ZibbyKeaton
+- @williamkapke (William Kapke)
```
Comment (bnb) [inline meetings/2018-04-19.md]: And William is a member as well 👍

File: meetings/2018-04-19.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+
+## Agenda
+
+*Extracted from **cc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+- [2018-04-27 Public User Feedback Meeting - Enterprise Use Case](https://github.com/nodejs/user-feedback/issues/52)
+- [Travel Fund approval and being careful about funds available](https://github.com/nodejs/admin/issues/99)
+- [Add Node Slack Community to scope](https://github.com/nodejs/admin/pull/49)
+
+### Tabled for next meeting...
```
Comment (bnb) [inline meetings/2018-04-19.md]: Side note, this is a really stellar structure. Love it, Jem!

File: meetings/2018-04-19.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+### Couple of final notes related to mentorship before public session expires: 
+
+- Group does need your input!
+- If you need team members, mentees could be good candidates
+- everyone should specifically check out the last task (Create a mentorship subgroup focused on to identify 6 month projects in Node.js [#30](https://github.com/nodejs/mentorship/issues/30))
+- Add suggestions to the issue or join the meeting to discuss this
+
+Ran out of time to address further issues for now. 
+
+**Public session timeboxed for 30 minutes followed with to a private meeting**
```
Comment (bnb) [inline meetings/2018-04-19.md]: > followed with to a private

Small typo: Remove `to`

**Comment by JemBijoux** [comment]
😅 Having everyone fill in their names, I got complacent about a couple of those labels. Thanks for the careful eye @bnb.

**Comment by bnb** [comment]
LGTM. Merging!

**Pull request merged**